src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilterTest.java

```diff
@@ -481,9 +479,28 @@ public void testTagsAreClearedEachRun() {
         final TreeWalkerAuditEvent dummyEvent2 = new TreeWalkerAuditEvent(contents2, "filename",
                 new LocalizedMessage(1, null, null, null, null, Object.class, null), null);
         suppressionCommentFilter.accept(dummyEvent2);
-        final List<SuppressionCommentFilter.Tag> tags =
-                Whitebox.getInternalState(suppressionCommentFilter, "tags");
+        final List<?> tags = Whitebox.getInternalState(suppressionCommentFilter, "tags");
         assertEquals("Invalid tags size", 1, tags.size());
     }
 
+    /**
+     * Returns an instance of {@link SuppressWithNearbyCommentFilter} internal type {@code Tag}.
+     * Testing implementation details is a bad practice, but we also need 100% test coverage.
+     * For this reason, it is necessary to use reflection to gain access to the inner field here.
+     *
+     * @return {@code Tag} instance
+     */
+    private static Object getTag() {
+        final SuppressWithNearbyCommentFilter suppressionCommentFilter =
+                new SuppressWithNearbyCommentFilter();
+        final FileContents contents =
+                new FileContents("filename", "//SUPPRESS CHECKSTYLE ignore");
+        contents.reportSingleLineComment(1, 0);
+        final TreeWalkerAuditEvent dummyEvent = new TreeWalkerAuditEvent(contents, "filename",
+                new LocalizedMessage(1, null, null, null, null, Object.class, null), null);
+        suppressionCommentFilter.accept(dummyEvent);
```

Comment:
Some of this code is similar to the code in `testTagsAreClearedEachRun`.
Does it make sense to make another util method of the common that would allow us to do that kind of stuff for tests?

Reply:
done, thank you. Now it looks more consistent.